Source/BottomPager/YPBottomPagerView.swift

```diff
@@ -27,7 +27,7 @@ final class YPBottomPagerView: UIView {
             0,
             |scrollView|,
             0,
-            |header| ~ 50
+            |header| ~ 44
```

Comment:
👍

Reply:
It should be 50 - i've compared with Insta

Reply:
Yes perfect :)